### ISSUE TITLE
Create (Unit) Tests

### ISSUE THREAD

**Issue by daknob** [DaKnOb/TorPaste#42]
It is very important to have tests written for TorPaste. Currently this has been ignored and manual testing was used instead, but the process has already caused some issues. 

Which testing frameworks do you use for Python? Have you used Flask's testing framework?

**Comment by j11e** [comment]
[http://flask.pocoo.org/docs/0.11/testing/](http://flask.pocoo.org/docs/0.11/testing/) is probably a good starting point for research.

**Comment by abit2** [comment]
hey there i would to work on this

**Comment by j11e** [comment]
I don't see a problem with that! Just please keep us updated regarding your choices (testing framework, what you test, etc.), so that we can make sure we don't forget anything. Discussing before coding is better than asking you to change things when you send your pull request ;)

**Comment by daknob** [comment]
Exactly, this is always a good thing: to discuss how we're going to solve an issue before we write the code. Unfortunately I think GitHub can only add repository owners as assignees, so I'm not quite sure if I can add you as one. In any case, keep us posted on your decisions, and we can discuss them together.
